### ISSUE TITLE
Java 9 Compliance (Illegal reflective access warnings)

### ISSUE THREAD

**Issue by clavin-xlnx** [Xilinx/RapidWright#4]
Turns out RapidWright is relying on packages that are not fully Java 9 compliant and are using APIs that are deprecated and will no longer be usable in future releases.  One example of the packages causing the issues:

WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by jnr.posix.JavaLibCHelper (file:/proj/xsjhdstaff2/clavin/workspace/RapidWright/jars/jython-standalone-2.7.0.jar) to method sun.nio.ch.SelChImpl.getFD()
WARNING: Please consider reporting this to the maintainers of jnr.posix.JavaLibCHelper
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

**Comment by clavin-xlnx** [comment]
Looks like this one might hang around for a while.  Jython has been trying to purge these exceptions but there is one use case that has proven difficult: http://bugs.jython.org/issue2656.

**Comment by HarrisonMc555** [comment]
I similarly get an "illegal reflective access" error when running several RapidWright programs (I think all the ones I've tried so far), but from a different source.

```
$ java com.xilinx.rapidwright.device.browser.DeviceBrowser
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.caucho.hessian.io.Hessian2Input (file:/.../RapidWright/RapidWright/jars/hessian-4.0.6.jar) to field java.lang.Throwable.detailMessage
WARNING: Please consider reporting this to the maintainers of com.caucho.hessian.io.Hessian2Input
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

It appears every time I run code, including those intended to be run from the command line (e.g. `SLRCrosserGenerator`, although I know that's just an example). This can clutter the output of these tools and be fairly distracting.

The previous comment was from Jython, but this one is Hessian, so I figured I would list it here for the sake of completeness. Is there a way we can avoid the illegal access or to silence those warnings?

**Comment by clavin-xlnx** [comment]
I have removed any use of the Hessian library in 2018.3.0, and it is removed from the rapidwright_jars.zip in 2018.3.1.  

It seems that Kryo still trips this message and it can't be turned off without a JVM flag (see [this bug](https://github.com/EsotericSoftware/kryo/issues/626)). I may have to modify their library to avoid the pesky messages in Java 9, 10, 11.  

Jython seems to have resolved the [issue as well](https://bugs.jython.org/issue2656), but they have yet to make a release that includes the fix.  I may custom build their current working set if it takes too long.

**Comment by clavin-xlnx** [comment]
This issue should be resolved in the [latest release (2019.2.2)](https://github.com/Xilinx/RapidWright/releases/tag/v2019.2.2-beta).
